README.md

```diff
@@ -33,6 +33,26 @@ If you want to build KEVM and KLab from source, you can:
     make kevm -j3
     ```
 
+### Setup Solidity Compiler and Dapp Tools
+
+First you'll need to install the [Nix packager](https://nixos.org/download.html).
+
+```sh
+curl -L https://nixos.org/nix/install | sh
+```
+
+Then you can install the Solidity compiler:
+
+```sh
+nix-env -f https://github.com/dapphub/dapptools/archive/master.tar.gz -iA solc-static-versions
+```
+
+And finally, install the [DappHub toolkit](https://dapp.tools/).
+
+```sh
+curl https://dapp.tools/install | sh
+```
+
```

Comment:
These should go at the top of `Instalation & Running`.

Reply:
Good idea, fixed.